lib/spack/spack/ci/__init__.py

```diff
@@ -451,18 +451,17 @@ def get_unaffected_pruners(
 
         affected_pkgs.update(repo_affected_pkgs)
 
+    if not affected_pkgs:
+        tty.info("Skipping unaffected pruning no package changes were detected")
```

Comment:
```suggestion
        tty.info("Skipping unaffected pruning: no package changes were detected")
```

maybe fix the wording while you're at it